layouts/_default/sitemap.xml

```diff
@@ -0,0 +1,28 @@
+{{/* This template is overridden to emit a versionless permalink for the most recent version. This allows us to submit the sitemap to crawlers with true canonical URLs. */}}
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9"
+  xmlns:xhtml="http://www.w3.org/1999/xhtml">
+  {{ range .Data.Pages }}
+  <url>
+    {{- $permalink := .Permalink }}
+    {{- if ($.Site.Params.section.versions) }}
+      {{- if (eq (index $.Site.Params.section.versions 1) $.Site.Params.section.version) }}
+        {{- $permalink = .Permalink | replaceRE `(http[s]*:\/\/([^\/]*)\/manual\/)([^\/]*\/)(.*)` `$1$4` }}
+      {{- end }}
+    {{- end }}
+    <loc>{{ $permalink }}</loc>{{ if not .Lastmod.IsZero }}
+    <lastmod>{{ safeHTML ( .Lastmod.Format "2006-01-02T15:04:05-07:00" ) }}</lastmod>{{ end }}{{ with .Sitemap.ChangeFreq }}
+    <changefreq>{{ . }}</changefreq>{{ end }}{{ if ge .Sitemap.Priority 0.0 }}
+    <priority>{{ .Sitemap.Priority }}</priority>{{ end }}{{ if .IsTranslated }}{{ range .Translations }}
+    <xhtml:link
+                rel="alternate"
+                hreflang="{{ .Lang }}"
+                href="{{ .Permalink }}"
```

Comment:
Note that I did not replace these `.Permalink`s with `$permalink` because I think they're referring to something different -- the permalink of a translated page. I don't think it matters, because we don't support multiple languages.